README.md

```diff
@@ -124,42 +133,47 @@ The config flow will then continue to ask you if you want to put your Smile and
 
-# Smile?
+## Why 'Smile'?
 
 We use the term Smile for the 'device connected to your home network', called Smile P1 for their power-meter, Smile if you have an Anna or Adam.
 
-# Testing
+## Is it tested?
 
 While we try to make sure that everything works as intended, we can't really test out changes happening to hardware devices. Our testing is done through testing against files from community members (see [python-plugwise tests](https://github.com/plugwise/python-plugwise/tree/main/tests)) and if you have a setup you are willing to share we highly welcome that. Just send us the files or submit a PR. Including your test code into the `tests/test_Smile.py` code is highly recommended.
 
 Results of our tests are checked by Travis, click the left button (the one that should say 'Build passing' :)) on the [python-plugwise repository](https://github.com/plugwise/python-plugwise/).
 
-# ~~There is Anna support in HA Core already~~ Replaced by the new Plugwise component, based on this beta-version
+## There is Plugwise / used to be Anna support in HA Core already?
+
+**The former 'anna' support was replaced by the new Plugwise component, based on this beta-version.**
 
-From the original sources by @laetificat it was improved upon and upstreamed by @CoMPaTech. Right after that @bouwew joined to improve and help maintain the code. As of 2020 @brefra joined so we have a full range of Plugwise products supported.
+From the original sources by @laetificat it was improved upon and upstreamed by @CoMPaTech for Anna. Right after that @bouwew joined to improve and help maintain the code - as a result also Adam and P1 became supported. As of 2020 @brefra joined for the USB part(s) so we have a full range of Plugwise products supported.
 
-As things like async were in high demand from HA Core, desired by the original author and a great challenge for us we rewrote it largely. The Plugwise Smile Beta repository (accompanying the Plugwise-Smile python module) is intended for development purposes, just as `anna-ha` was for `haanna` (respectively the original before upstreaming and original python module).
+As things like async were in high demand from HA Core, desired by the original author and a great challenge for us we rewrote it largely. The Plugwise Beta repository (accompanying the Plugwise python module) is intended for development purposes, just as `anna-ha` was for `haanna` (respectively the original before upstreaming and original python module). Note that 'Plugwise-Smile' existed for a while before @brefra joined.
 
 With the three combined forces we now support, maintain and improve on:
 
 - `plugwise-beta` (this repository) for beta-testing new features to go into the `plugwise`-integration for HA
-- `python-plugwise` for connectivity with
+- [`python-plugwise`](https://github.com/plugwise/python-plugwise) for connectivity with
   - Smile (Adam, Anna and P1) and Stretch products (i.e. the new Plugs)
```

Comment:
Plugs should be linked to Adam only, the Stretches work with the Circles, Stealths etc, the same devices that connect to the Stick.

Reply:
Instead of re-writing what we have, just removed the indents. In that case leaving
`- [`python-plugwise`](https://github.com/plugwise/python-plugwise) for connecting to Plugwise products`
so we can just have it in that repo to describe what it does

Reply:
Fine by me :)